### DISCUSSION TITLE
Развитие механических возможностей медицины

### DISCUSSION THREAD

**Discussion by CaiDus** [octothorp-team/public-dobrograd#724] [Предложения]
Иногда ты по собственной глупости или специально падаешь с крыши, тебя поднимают медики отыгрывают все, тебя спасли но только формально, ты досих пор мертв и тебе остается ждать. И за отсутствия механической возможности воскрешать медикам нет смысла отыгрывать что ты все еще жив. Я предлагаю увеличить функционал медиков, к примеру в карете мед помощи есть передвижная палата и можно добавить возможность сажать на нее игроков и увозить.  Добавить в больнице способ воскрешать людей путем рп, а не надеяться что ответит админ и тебя воскресит(а ему нужно еще все доказать).

**Comment by SemjonBelus** [comment]
Пока я был в DMC, то такая-же идея была и у меня.. Но Sam Fisher меня переубедил
![image](https://user-images.githubusercontent.com/93833754/141641983-69e181b9-3770-42a5-a70c-4806c22efc4d.png)
![image](https://user-images.githubusercontent.com/93833754/141642046-df4236e8-5386-45ef-9834-1ebff3efa663.png)
Самуэль сказал довольно умную мысль когда-то давно, которая до сих пор актуальна. Пока нет системы персонажей: я против.
